cucumber/cucumber-tests.ts

```diff
@@ -1,6 +1,8 @@
-/// <reference types="assert" />
 import cucumber = require("cucumber");
 
+var assert = {deepEqual: function (a: Object, b: Object) {
```

Comment:
Why this change? The tests won't actually run, just be compiled. So why not use the already existing definition?